src/widgets/crosshairs.js

```diff
@@ -0,0 +1,104 @@
+import { vec3 } from 'gl-matrix';
+import vtkCrosshairsWidget from '@/src/vtk/CrosshairsWidget';
+import {
+  onViewMouseEvent,
+  onAddedToView,
+  onWidgetStateChanged,
+  onBeforeDelete,
+} from './context';
```

Comment:
@floryst do you plan to add more hooks to `./context`?

 I want to unset "placed" when another image is selected, but none of these hooks (or the others in `./context`) fire when that happens.